src/index.html

```diff
@@ -8,6 +8,34 @@
     <link rel="stylesheet" href="./styles/index.scss">
   </head>
   <body>
-    <h1>Product cards</h1>
+    <article class="card" data-qa="card">
+      <img src="Rectangle 4.png"
+      class="card__img"
+      alt="picture"
+      >
```

Comment:
```suggestion
      <img
        src="Rectangle 4.png"
        class="card__img"
        alt="picture"
      >
```
if there are 3 or more attributes :arrow_up: